components/src/core/Drawer/Drawer.tsx

```diff
@@ -238,6 +249,7 @@ export const DrawerComponent: React.FC<DrawerComponentProps> = (props) => {
                     <div
                         className={clsx(defaultClasses.smooth, classes.smooth)}
                         style={{ width: containerWidth, height: '100%' }}
+                        id={'pxblue-drawer-content-width'}
```

Comment:
shall we prefix/namespace these fixed ids in some way to ensure they don't get stomped on (unlikely, but I'm just wondering), e.g.,:
```
id={'@@pxblue/drawer-content-width'}
```

Reply:
Good idea, I updated the ID used.